irohad/torii/impl/command_service.cpp

```diff
@@ -249,4 +203,59 @@ namespace torii {
     return grpc::Status::OK;
   }
 
+  void CommandService::checkCacheAndSend(
+      boost::optional<iroha::protocol::ToriiResponse> &resp,
+      grpc::ServerWriter<iroha::protocol::ToriiResponse> &response_writer)
+      const {
+    if (resp) {
+      if (resp->tx_status()
```

Comment:
So, can we combine `or` function for statuses in a functional way?

Reply:
It was really stupid from my side to introduce isFinalStatus() function and do not use it here.